src/js/templates/maps/layer-category-item.html

```diff
@@ -1,13 +1 @@
-<div class="<%= classNames.metadata %>">
-  <span class="<%= classNames.icon %>">
-    <svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24"><path d="m3.2 7.3 8.6 4.6a.5.5 0 0 0 .4 0l8.6-4.6a.4.4 0 0 0 0-.8L12.1 3a.5.5 0 0 0-.4 0L3.3 6.5a.4.4 0 0 0 0 .8Z"/><path d="M20.7 10.7 19 9.9l-6.7 3.6a.5.5 0 0 1-.4 0L5 9.9l-1.8.8a.5.5 0 0 0 0 .8l8.5 5a.5.5 0 0 0 .5 0l8.5-5a.5.5 0 0 0 0-.8Z"/><path d="m20.7 15.1-1.5-.7-7 3.8a.5.5 0 0 1-.4 0l-7-3.8-1.5.7a.5.5 0 0 0 0 .9l8.5 5a.5.5 0 0 0 .5 0l8.5-5a.5.5 0 0 0 0-.9Z"/></svg>
```

Comment:
we should preserve the default value for layer category

Reply:
I guess we missed a unit test. Combining this with your comment above about simplifying the template and the way icons are rendered. I'll svg icon if isSvgIcon otherwise icon as a font awesome icon if it is defined, otherwise, render the default SVG